xtask/src/main.rs

```diff
@@ -2,6 +2,7 @@ use clap::{Parser, Subcommand};
 
 mod publish;
 mod runchecks;
+pub(crate) mod utils;
```

Comment:
Can we write `mod utils` and import things as `use crate::crate_name::function`?